lib/github.js

```diff
@@ -71,6 +71,12 @@ Organization.prototype.getRepositories = function(callback) {
             cli.debug('next page ' + this.nextPageUrl);
             cli.debug('last page ' + this.getLastPage());
 
+            if ( body.message ) {
```

Comment:
If there is some error, github returns json with only two fields: `message` and `documentation_url`. Currently `cloneorg` doesn't check for these and thinks, that the organization repo list is just empty. With this PR the exact error message GitHub returns is shown to user.